packages/shared-api/lib/db/lambda/migrator.ts

```diff
@@ -0,0 +1,17 @@
+import { exec } from "node:child_process";
+import { promisify } from "node:util";
+
+import { getConnectionString } from "../connection";
+
+export const handler = async (event: { schema: "api" | "auth" }) => {
```

Comment:
Remove (schema: "api" | "auth") typing, as this causes a conceptual circular dependency. Just keep the parameter untyped.